frontend/components/FieldImage/FieldImage.jsx

```diff
@@ -9,27 +10,54 @@ import Label from 'components/Label/Label'
 
 export default class FieldImage extends Component { 
 
+  static propTypes = {
+
+    maxWidth: proptypes.number,
```

Comment:
Do we need `maxWidth` as a prop? We talked about all field components receiving the exact same props, so it's easy to make their loading dynamic.

If it does need it, can you add documentation to the prop?